kpops/components/base_components/helm_app.py

```diff
@@ -141,6 +141,9 @@ def template_flags(self) -> HelmTemplateFlags:
             api_version=self.config.helm_config.api_version,
         )
 
+    def helm_values(self) -> dict[str, Any] | None:
```

Comment:
imo the wrapper method is unnecessary, why not just call `helm.get_values` directly?

Reply:
Yeah, I know I will have it removed 😄